### PR TITLE
Fix broken saving of svg in Chrome >=60

### LINKED ISSUE #193: Chrome/Opera browser not save, export.
I try to export or save image but not work .
The result of this is a redirect to a new tab web page : about:blank

### PR DESCRIPTION
**Pull request by TeemuKoivisto**
Fixes #193 which was due to https://stackoverflow.com/questions/45603201/window-is-not-allowed-to-navigate-top-frame-navigations-to-data-urls

I don't know if there is something else also broken but this fixes my use case so I don't have a lot of motivation to spend more time on this 😃

### CODE REVIEW

File: editor/svg-editor.js [hunk 1]
```diff
@@ -3762,6 +3767,7 @@ TODOS
 			};
 
 			var saveSourceEditor = function() {
+				debugger
```
Comment (codedread) [inline editor/svg-editor.js]: Remove

File: editor/svg-editor.js [hunk 2]
```diff
@@ -1049,8 +1049,13 @@ TODOS
...
 
-				// Opens the SVG in new window
-				var win = wind.open('data:image/svg+xml;base64,' + Utils.encode64(svg));
+				// Since saving SVGs by opening a new window was removed in Chrome use artificial link-click
+				// https://stackoverflow.com/questions/45603201/window-is-not-allowed-to-navigate-top-frame-navigations-to-data-urls
+				var a  = document.createElement('a');
+				a.href = 'data:image/svg+xml;base64,' + Utils.encode64(svg);
+				a.download = 'icon.svg';
+		
+				a.click()
```
Comment (codedread) [inline editor/svg-editor.js]: Please end in a semicolon

**Comment by TeemuKoivisto** [comment]
@codedread Done

File: editor/svg-editor.js [hunk 3]
```diff
@@ -1049,8 +1049,13 @@ TODOS
 					return;
 				}
 
-				// Opens the SVG in new window
-				var win = wind.open('data:image/svg+xml;base64,' + Utils.encode64(svg));
+				// Since saving SVGs by opening a new window was removed in Chrome use artificial link-click
+				// https://stackoverflow.com/questions/45603201/window-is-not-allowed-to-navigate-top-frame-navigations-to-data-urls
+				var a  = document.createElement('a');
+				a.href = 'data:image/svg+xml;base64,' + Utils.encode64(svg);
+				a.download = 'icon.svg';
```
Comment (codedread) [inline editor/svg-editor.js]: Ideally this should use the title of the document (.svg), but I think this is good enough for now.

**Pull request merged**